src/Swift.Bindings/src/Emitter/StringEmitter/Handler/MethodHandler.cs

```diff
@@ -665,6 +735,23 @@ private void EmitGenericArguments(CSharpWriter csWriter)
             csWriter.WriteLine();
         }
 
+
+        private void EmitProtocolWitnessTables(CSharpWriter csWriter)
+        {
+            foreach (var genericParameter in _env.MethodDecl.GenericParameters)
+            {
+                var placeholderName = _env.GenericTypeMapping[genericParameter.TypeName].PlaceholderName;
```

Comment:
Naming nit: Placeholder is confusing (at least to me). In C# these are called type parameters, or generic parameters. I've never heard the term "placeholder".
Is there a reason we chose this name?
The official docs seem to use "type parameter": https://docs.swift.org/swift-book/documentation/the-swift-programming-language/genericparametersandarguments/#Generic-Parameter-Clause

Reply:
No big reason. It used to be `TypeName` in the past https://github.com/dotnet/runtimelab/pull/2946/commits/eec9e3363fad28bb9d5da92e534b1f660716866e#diff-0d8888e3e8f8b2faf5860214283c2e3829a7f8668153de57ecbeb1ec9b7cd1d6

I think we could change to `typeParamaterName` or even `csTypeParamName` to indicate that this is a `c#` translation of the type parameter. 

This is valid swift 😅 
```swift
struct 📦<🎁> {
    var item: 🎁
}
```

Reply:
> This is valid swift

1 - ROTFL
2 - OK, but why should we care? (as in, I won't lose any sleep if we don't support this anytime in this decade)

Reply:
This is just an extreme example to show that swift has much more freedom in naming stuff. Less extreme example would be type param in swift being keyword in c#. I do not think we need to outright guarantee support for such names in the projection tooling, but it is good to remember that we need to sanitize/translate any typename before using it in projections.